core/templates/pages/story-editor-page/modal-templates/story-editor-unpublish-modal.component.html

```diff
@@ -1,18 +1,193 @@
-<div class="modal-header">
-  <p>
-    Are you sure you want to unpublish this story? Unpublishing a story will
-    delete any corresponding translation opportunities from the Contributor
-    Dashboard and reject any related pending translation suggestions.
-  </p>
+<div *ngIf="platformFeatureService.status.SerialChapterLaunchCurriculumAdminView.isEnabled">
+  <div class="modal-header">
+    <p>
+      You're attempting to unpublish chapters, please specify a reason and note the warning
+      message before proceeding.
+    </p>
+  </div>
+  <div class="modal-body">
+    <div class="reason-select">
+      <span>Select Reason:</span>
+      <div ngbDropdown class="oppia-unpublishing-dropdown">
+        <div ngbDropdownToggle
+             aria-label="Dropdown toggle to select unpublishing reason"
+             aria-haspopup="menu"
+             aria-expanded="false"
+             class="selected-reason-text">
+          {{ selectedReasonText }}
+        </div>
+        <div ngbDropdownMenu
+             class="oppia-unpublishing-dropdown-menu">
+          <div ngbDropdownItem
+               (click)="setReason('BAD_CONTENT')"
+               class="oppia-unpublishing-dropdown-item"
+               tabindex="0">
+            {{ badContentReasonText }}
+          </div>
+          <hr>
+          <div ngbDropdownItem
+               (click)="setReason('CHAPTER_NEEDS_SPLITTING')"
```

Comment:
Also can you please add: 
- `(click)="setReason('CHAPTER_NEEDS_SPLITTING')"`

to also make is accessible when user is uisng the keyboard.

Reply:
Done